### PR TITLE
Feature/initial configuration

### PR DESCRIPTION
**Pull request by rvillaab**

### CODE REVIEW

File: pckg/service/questionService.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+	return QuestionServiceImpl{DB: dB}
+}
+
+func (qstImpl QuestionServiceImpl) CreateQuestion(question ent.Question) (ent.Question, error) {
+
+	question.CreateQuestion(qstImpl.DB)
+
+	return question, nil
+}
```
Comment (RicardoBeltranRamirez) [inline pckg/service/questionService.go]: `type QuestionServiceImpl struct {	
  dao QuestionDao
}`

`type QuestionDao struct {
	DB *sql.DB
}`


`func (qstImpl QuestionServiceImpl) CreateQuestion(question ent.Question) (ent.Question, error){
err := qstimpl.dao.CreateQuestion(question)	
if err != nil{
   return nil, err
}
return question, nil
}`

File: pckg/test/mocks/mockQuestionService.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	MockGetAllQuestionsFunc    GetAllQuestionsFunc
+	MockCreateQuestionFunc     CreateQuestionFunc
+	MockUpdateQuestionFunc     UpdateQuestionFunc
+	MockDeleteQuestionFunc     DeleteQuestionFunc
+	MockGetQuestionByIdFunc    GetQuestionByIdFunc
+	MockGetQuestionsByUserFunc GetQuestionsByUserFunc
+}
+
+func (m *MockQuestionService) SetGetAllFunc(f GetAllQuestionsFunc) {
+	m.MockGetAllQuestionsFunc = f
```
Comment (RicardoBeltranRamirez) [inline pckg/test/mocks/mockQuestionService.go]: `if m.MockGetAllQuestionsFunc != nil{
      return m.MockGetAllQuestionsFunc = f
}
return nil
´

**Review by RicardoBeltranRamirez — CHANGES_REQUESTED** [review]
Just a few comments

File: pckg/service/questionService.go (hunk 1 above)
Comment (rvillaab) [reply pckg/service/questionService.go]: Done!

File: pckg/test/mocks/mockQuestionService.go (hunk 2 above)
Comment (rvillaab) [reply pckg/test/mocks/mockQuestionService.go]: Done!

**Review by RicardoBeltranRamirez — APPROVED** [review]
Excellent! LGTM

**Pull request merged**